rts/at/bridge/src/main/java/org/jboss/narayana/rest/bridge/inbound/InboundBridgeManager.java

```diff
@@ -45,13 +45,9 @@
 
     private final Map<String, Xid> transactionXids;
 
-    public static InboundBridgeManager getInstance() {
+    public static synchronized InboundBridgeManager getInstance() {
```

Comment:
Isn't it just the initial check of INSTANCE that it would complain about - but I am OK with your change